frontend/src/services/FeatureFlagManager.ts

```diff
@@ -6,6 +6,7 @@ import { NextRequest, NextResponse } from "next/server";
 
 import { CookiesStatic } from "js-cookie";
 import { ReadonlyRequestCookies } from "next/dist/server/web/spec-extension/adapters/request-cookies";
+import { ServerSideSearchParams } from "../types/searchRequestURLTypes";
```

Comment:
out of scope comment. the mixing of relative paths gave me pause. i created a ticket to address for the frontend: https://github.com/HHS/simpler-grants-gov/issues/1868 , so this is fine for now.

Reply:
Sure, we can take a look at making imports more consistent